integration/util/download.js

```diff
@@ -19,24 +19,46 @@ function download(url, path, options = {}, handle = null) {
     ...options,
   };
   return new Promise(async (resolve, reject) => {
-    let f = handle || await fs.open(path, 'w');
     let bail = false;
+    let f_;
```

Comment:
Here we properly handle the case where we get a 404 (we used to write the 404 message to the file)